app/serializers/api/admin/for_order_cycle/supplied_product_serializer.rb

```diff
@@ -19,13 +19,7 @@ def master_id
         end
 
         def variants
-          variants = if order_cycle.present? &&
-                        order_cycle.prefers_product_selection_from_coordinator_inventory_only?
```

Comment:
This conditional is deleted without replacement. Is that logic not needed? Could you explain it a bit?

Reply:
it is. If you go commit by commit you'll see. However, it was a rough attempt and I need to see if this is enough or not.